project/PekkoDependency.scala

```diff
@@ -0,0 +1,125 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * license agreements; and to You under the Apache License, version 2.0:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * This file is part of the Apache Pekko project, derived from Akka.
+ */
+
+/*
+ * Copyright (C) 2017-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+import sbt._
+import Keys._
+
+import scala.util.matching.Regex.Groups
+
+object PekkoDependency {
```

Comment:
I am assuming you copied this from pekko-http? Don't have a problem with this now but I think that this solution is a bit overengineered (however it does serve our purposes due to everything being released as a snapshot).

Reply:
yes - copied from pekko-http - I think it will do until we get a release out and then we'll probably not tie ourselves to snapshot versions at all